optuna_dashboard/__init__.py

```diff
@@ -13,6 +13,8 @@
 from ._named_objectives import set_objective_names  # noqa
 from ._note import get_note  # noqa
 from ._note import save_note  # noqa
+from ._streamlit_helper import render_trial_note  # noqa
+from ._streamlit_helper import render_user_attr_form_widgets  # noqa
```

Comment:
Could you add the `optuna.streamlit` package and put these functions in it?

Reply:
OK. Does you mean I create directory "optuna-dashboard/streamlit" and put my code in this dir instead of "optuna-dashboard/optuna-dashboard/_streamlit_helper.py"?

Reply:
Exactly 👍